### ISSUE TITLE
Would you like to cross platform?

### ISSUE THREAD

**Issue by ShortArrow** [chocolatey/choco#1854]
powershell is now available cross platform. So the question is. Do you have plans to make it available on MacOS or Linux? Or is it worthwhile to proceed with such development?

It became an issue when sharing information for building a development environment with a team of mixed Mac and Windows users.

I love Chocolatey because I'm a Windows user, but it doesn't help when I'm building a Mac environment for a junior Mac user. The second adventure of environmental construction begins. This is not efficient.

I think that more users will spread if it can be used cross platform. Homebrew has also made inroads into Windows with the advent of WSL2. If it is this, the day when chocolate melts may be near.

I am inexperienced with C # and I am not confident to contribute to development, but I hope to be cross-platform.

**Comment by ferventcoder** [comment]
@ShortArrow we've talked about it for years (we call it "single pane of glass"). There are quite a few technical hurdles and we really have a wide area of Windows we need to focus on solving first before we can look at any kind of cross platform use.

**Comment by pauby** [comment]
Thanks for raising this issue. After reviewing it, it's not something we expect to prioritise in the foreseeable future. We’re closing it to keep our backlog focused, but we’re open to revisiting it in the future if priorities shift or new interest arises. You are welcome to comment here or open a new issue if you'd like to discuss it further down the line.
